src/Stores/TaskStore.ts

```diff
@@ -0,0 +1,48 @@
+import { action, computed, makeObservable, observable } from "mobx";
+import TaskItem from "../Models/TaskItem";
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
This is not used anyware, should not be commited.

Reply:
It is removed and fixed